_pages/known-issues/hosted/unchained.md

```diff
@@ -16,7 +16,7 @@ As Trezor is stateless, these steps will only confirm that `1` of your keys is p
  [`3-of-5` would be better](/quorum-advanced), especially when using a hosted service that already:
 
 * Abstracts away a lot of multisig's complexity
-* Holds `1` (or more) of your seeds
+* Holds 1 (or more) of your seeds
```

Comment:
The convention in the guide has been to use `code` formatting for `m-of-n`, do you think that's bad? I like it, but consistency is probably the most important. IThere are other examples that show it as `1` above on this very page.

Reply:
I do like the `code` formatting for quorum descriptions (e.g. `2-of-3`). I'm on the fence about using it for individual numbers but, regardless, I agree consistency is more important and that convention is used elsewhere (I just didn't register the few other uses).

PR incoming...